git/objects/commit.py

```diff
@@ -738,3 +739,24 @@ def _deserialize(self, stream: BytesIO) -> "Commit":
         return self
 
     # } END serializable implementation
+
+    @property
+    def co_authors(self) -> List[Actor]:
+        """
+        Search the commit message for any co-authors of this commit.
+        Details on co-authors: https://github.blog/2018-01-29-commit-together-with-co-authors/
+
+        :return: List of co-authors for this commit (as Actor objects).
+        """
+        co_authors = []
+
+        if self.message:
+            results = re.findall(
+                r"^Co-authored-by: ((?:\w|\-| ){0,38}) <(\S*)>$",
```

Comment:
For consistency and probably correctness, one could either use [this regex](https://github.com/gitpython-developers/GitPython/blob/main/git/util.py#L690:L690) or instantiate an actor [like this](https://github.com/gitpython-developers/GitPython/blob/main/test/test_actor.py#L13:L13).

Reply:
Sure thing! I kept the same regex, to ensure that the correct prefix is used, and then instantiated an Actor as requested.